guides/content/developer/tutorials/getting_started_tutorial.md

```diff
@@ -9,19 +9,27 @@ Before starting this tutorial, make sure you have Ruby and RubyGems installed on
 
 By following this tutorial, you will create a simple Spree project called `mystore`. Before you can start building the application, you need to make sure that you have Rails itself installed.
 
-To run Spree 3.1 you need the latest Rails version, 4.2.6.
+To run Spree 3.1 you need the latest Rails 4 version, which is currently version
+4.2.7.1.
+
+The latest version of Spree in development is Spree 3.2 RC2. It requires Rails 5
```

Comment:
RC2 is really stable and is used on production :) We at Spark use it in all of our new projects